odl/tomo/backends/astra_cuda.py

```diff
@@ -39,12 +39,7 @@ class AstraCudaProjectorImpl(object):
 
     """Thin wrapper around ASTRA."""
 
-    algo_id = None
-    vol_id = None
-    sino_id = None
-    proj_id = None
-
-    def __init__(self, geometry, reco_space, proj_space, use_cache):
+    def __init__(self, geometry, reco_space, proj_space):
```

Comment:
Yeah, kind of an obvious change. Why should the wrapper whose purpose is caching ask again if it should cache? Basically it was asking if it was allowed to stay alive after usage, weird :-)
Nice catch anyway.